twilight-model/src/guild/auto_moderation/trigger_metadata.rs

```diff
@@ -23,6 +23,9 @@ pub struct AutoModerationTriggerMetadata {
     /// [Discord Docs/Keyword Matching Strategies]: https://discord.com/developers/docs/resources/auto-moderation#auto-moderation-rule-object-keyword-matching-strategies
     #[serde(skip_serializing_if = "Option::is_none")]
     pub presets: Option<Vec<AutoModerationKeywordPresetType>>,
+    /// Whether to automatically detect mention raids.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub mention_raid_protection_enabled: Option<bool>,
```

Comment:
This is not nullable?

Reply:
All of these fields in this struct are documented as non-nullable. However, they are all marked as nullable here because this struct is used for patching.